src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchElementViewModel.cs

```diff
@@ -19,17 +20,25 @@ public class PackageManagerSearchElementViewModel : BrowserItemViewModel
         public ICommand DownvoteCommand { get; set; }
         public ICommand VisitSiteCommand { get; set; }
         public ICommand VisitRepositoryCommand { get; set; }
+        public DelegateCommand DownloadLatestToCustomPathCommand { get; set; }
 
+        public PackageManagerSearchViewModel PackageManagerSearchViewModel { get; private set; }
```

Comment:
This doesn't need to be a property, maybe turn it into a class data member. By the way, I'm wondering if we can avoid adding another parameter to the constructor of `PackageManagerSearchElementViewModel`. Can we not expose a new `OnRequestShowFileDialog` event from it directly and have the view subscribed to that to bring up a dialog? Just curious.